src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -126,7 +126,7 @@ public function reverseTransform($value)
 
         try {
             // read timestamp into DateTime object - the formatter delivers in UTC
-            $dateTime = new \DateTime(sprintf('@%s UTC', $timestamp));
+            $dateTime = new \DateTime(sprintf('@%s', $timestamp), new \DateTimeZone('UTC'));
```

Comment:
After bench, the usage of a DateTimeZone is slower 👎 

But the timestamp with the timezone  `parse_date("@1234 UTC")` indicate a warning : `Double timezone specification`. I removed the UTC specification and this time it's more faster! 👍